apps/landing/src/pages/team.tsx

```diff
@@ -1,8 +1,182 @@
+	},
+	{
+		name: 'Haden Fletcher',
+		role: 'Engineer & Designer',
+		image: 'team/haden.jpg',
+		socials: {
+			twitter: 'https://twitter.com/heymaxichrome',
+			twitch: 'https://twitch.tv/maxichrome',
+			github: 'https://github.com/maxichrome'
+		}
+	},
+	{
+		name: 'Benjamin Akar',
+		role: 'Engineer & Designer',
+		image: 'team/benja.jpg',
+		socials: {
+			twitter: 'https://twitter.com/benjaminakar',
+			twitch: 'https://twitch.tv/akawr',
+			github: 'https://github.com/benja'
+		}
+	},
+	{
+		name: 'Haris Mehrzad',
+		role: 'Engineer Intern',
+		image: 'team/haris.jpg',
+		socials: {
+			twitter: 'https://twitter.com/xPolarrr',
+			github: 'https://github.com/xPolar'
+		}
+	}
+];
+
+const investors: Array<TeamMemberProps> = [
+	{
+		name: 'Joseph Jacks',
+		role: 'Founder, OSSC',
+		investmentRound: 'Lead Seed',
+		image: 'investors/josephjacks.jpg'
+	},
+	{
+		name: 'Guillermo Rauch',
+		role: 'CEO, Vercel',
+		investmentRound: 'Co-Lead Seed',
+		image: 'investors/guillermo.jpg'
+	},
+	{
+		name: 'Naval Ravikant',
+		role: 'Founder, AngelList',
+		investmentRound: 'Co-Lead Seed',
+		image: 'investors/naval.jpg'
+	},
+	{
+		name: 'Neha Narkhede',
+		role: 'Confluent, Apache Kafka',
+		investmentRound: 'Seed',
+		image: 'investors/neha.jpg'
+	},
+	{
+		name: 'Austen Allred',
+		role: 'CEO, Bloom Institute of Technology',
+		investmentRound: 'Seed',
+		image: 'investors/austen.jpg'
+	},
+	{
+		name: 'Tom Preston-Werner',
+		role: 'Founder, GitHub',
+		investmentRound: 'Seed',
+		image: 'investors/TOM.jpg'
+	},
+	{
+		name: 'Tobias Lütke',
+		role: 'CEO, Shopify',
+		investmentRound: 'Seed',
+		image: 'investors/tobiaslutke.jpg'
+	},
+	{
+		name: 'Justin Hoffman',
+		role: 'Former VP Sales, Elasticsearch',
+		investmentRound: 'Seed',
+		image: 'investors/justinhoffman.jpg'
+	},
+	{
+		name: 'Ry Walker',
+		role: 'Founder, Astronomer',
+		investmentRound: 'Seed',
+		image: 'investors/rywalker.jpg'
+	},
+	{
+		name: 'Zachary Smith',
+		role: 'Head of Edge Infrastructure, Equinix',
+		investmentRound: 'Seed',
+		image: 'investors/zacharysmith.jpg'
+	},
+	{
+		name: 'Sanjay Poonen',
+		role: 'Former COO, VMware',
+		investmentRound: 'Seed',
+		image: 'investors/sanjay.jpg'
+	},
+	{
+		name: 'David Mytton',
+		role: 'CEO, console.dev',
+		investmentRound: 'Seed',
+		image: 'investors/davidmytton.jpg'
+	},
+	{
+		name: 'Peer Richelsen',
+		role: 'CEO, Cal.com',
+		investmentRound: 'Seed',
+		image: 'investors/peer.jpg'
+	},
+	{
+		name: 'Lester Lee',
+		role: 'Founder, Slapdash',
+		investmentRound: 'Seed',
+		image: 'investors/lesterlee.jpg'
+	},
+	{
+		name: 'Haoyuan Li',
+		role: 'Founder, Alluxio',
+		investmentRound: 'Seed',
+		image: 'investors/haoyuan.jpg'
+	},
+	{
+		name: 'Augusto Marietti',
+		role: 'CEO, Kong',
+		investmentRound: 'Seed',
+		image: 'investors/augusto.jpg'
+	},
+	{
+		name: 'Vijay Sharma',
+		role: 'CEO, Belong',
+		investmentRound: 'Seed',
+		image: 'investors/sharma.jpg'
+	},
+	{
+		name: 'Naveen R',
+		role: 'NocoDB',
```

Comment:
As suggested in #231.

```suggestion
		role: 'Founder, NocoDB',
```